modules/market/src/main/java/com/opengamma/strata/market/curve/IsdaCreditCurveDefinition.java

```diff
@@ -79,7 +79,7 @@
    * The nodes are used to find the par rates and calibrate the curve.
    */
   @PropertyDefinition(validate = "notNull")
-  private final ImmutableList<IsdaCreditCurveNode> curveNodes;
+  private final ImmutableList<? extends IsdaCreditCurveNode> curveNodes;
```

Comment:
We don't use `? extends` in bean fields. See `InterpolatedNodalCurveDefinition` and the `nodes` property which uses `builderType` to get this effect.